docs/content/2.guide/15.layers.md

```diff
@@ -32,8 +27,6 @@ Extending your project with a layer that contains locales can be done as follows
   export default defineNuxtConfig({
     extends: ['my-layer'],
     i18n: {
-      lazy: true,
-      langDir: './lang',
```

Comment:
In my testing, `langDir` still needs to be set! :thinking: 
@BobbieGoede

Reply:
Ah shoot, `langDir` has to be set when the project provides locales as well, but it's not required if a project without locales is only extending a layer configured with i18n. You're correct that in the example code the `langDir` should still be set, will get to fixing it!